vsts/vsts.yaml

```diff
@@ -245,7 +245,14 @@ jobs:
 
       - task: ComponentGovernanceComponentDetection@0
         displayName: "Component Detection"
-
+      
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@1
```

Comment:
Moved this to the top to avoid any possible tool interactions that might change the behavior.